libs/shared/ui/src/lib/components/icon/icon-awesome.enum.tsx

```diff
@@ -112,5 +112,6 @@ export enum IconAwesomeEnum {
   SHIELD_CHECK = 'icon-solid-shield-check',
   HOUSE = 'icon-solid-house',
   LOCK = 'icon-solid-lock',
+  WAND_MAGIC_SPARKLES = 'icon-solid-wand-magic-sparkles',
```

Comment:
you can remove it

Reply:
https://github.com/Qovery/console/pull/1265/commits/712b384bbe93abfa50ca67187be1e344a5a6bdcc